peregrine/resources/submission/graphql/node.py

```diff
@@ -525,7 +525,7 @@ def lookup_graphql_type(T):
     return {
         bool: graphene.Boolean,
         float: graphene.Float,
-        int: graphene.Int,
+        int: graphene.Float,
```

Comment:
could you update your PR description with more details on the issue you're fixing? this is more of an improvement than a new feature